forge/tests/forge/other/properties_undirected_tree.rkt

```diff
@@ -23,10 +23,13 @@ pred isUndirectedTree {
  {
      all m, n : Node | n->m in edges implies m->n in edges
  } 
- where {
+
+
+test suite for isUndirected {
 
         test expect {
             {isUndirected} is sat
+            {one Node} is sat // This should result in a warning but not an error.
```

Comment:
Does it really? It looks like everything is this `test expect` goes into one string for the check.